src/multirust-cli/common.rs

```diff
@@ -267,8 +267,15 @@ pub fn list_toolchains(cfg: &Cfg) -> Result<()> {
     if toolchains.is_empty() {
         println!("no installed toolchains");
     } else {
-        for toolchain in toolchains {
-            println!("{}", &toolchain);
+        if let Ok(Some(def_toolchain)) = cfg.find_default() {
+            for toolchain in toolchains {
+                println!("{}{}", &toolchain, if def_toolchain.name() == &*toolchain { " (default)" } else { "" });
```

Comment:
Can you please extract this if expression to a variable?

Can you add a test to `tests/cli-rustup.rs`? You can crib off of the test cases in `cli-v2.rs`. What we want is to verify that stdout contains a string like "nightly (default)".